tools/scaffolding/scaffolder.go

```diff
@@ -295,6 +306,8 @@ func main() {
 	defer os.RemoveAll(tmpout)
 	log.Println("Using tmpdir", tmpout)
 
+	helloWorldModule := string(reflect.Indirect(reflect.ValueOf(data)).FieldByName("HelloWorldModule").String())
```

Comment:
Found this way to access the data in the struct, not sure if there's a better way currently